searchcore/src/vespa/searchcore/proton/test/attribute_utils.h

```diff
@@ -18,7 +18,7 @@ struct AttributeUtils
         for (uint32_t i = 1; i < ia.getNumDocs(); ++i) {
             ia.update(i, value);
         }
-        ia.commit(lastSyncToken, lastSyncToken);
+        ia.commit(search::CommitParam(lastSyncToken, true));
```

Comment:
Why are we using _forceUpdateStats=true_ here? That is not equal to the previous behaviour. The same comment applies for other changes in this file.

Reply:
Removed semantic change